Src/Scripts/Include-GenerateUnitClassSourceCode.ps1

```diff
@@ -278,11 +280,30 @@ namespace UnitsNet
         public static $className Parse(string str, IFormatProvider formatProvider = null)
         {
             if (str == null) throw new ArgumentNullException("str");
-            string[] words = str.Split(new[] {" "}, StringSplitOptions.RemoveEmptyEntries);
-            if (words.Length < 2)
+
+            var numFormat = formatProvider != null ?
+                (NumberFormatInfo) formatProvider.GetFormat(typeof (NumberFormatInfo)) :
+                (NumberFormatInfo) CultureInfo.CurrentCulture.NumberFormat.Clone();
```

Comment:
I am actually not sure about this myself, but I think we should look at what double.Parse uses by default (considering the quantity is just forwarded there). Looking at [the docs](https://msdn.microsoft.com/en-us/library/fd84bdyt(v=vs.110).aspx) it looks like [NumberFormatInfo.CurrentInfo](https://msdn.microsoft.com/en-us/library/system.globalization.numberformatinfo.currentinfo(v=vs.110).aspx) is used which is essentially using CultureInfo.CurrentCulture.NumberFormat.

Do you think it would be better to switch to NumberFormatInfo.CurrentInfo?